File: codeforces/go/158A.go
package main

import "fmt"

func main() {
	var n, k, i, count int
	fmt.Scanf("%d %d\n", &n, &k)
	l := make([]int, n)
	for i = 0; i<n; i++ {
		fmt.Scan(&l[i])
	}
	for i = 0; i<n; i++ {
		if l[i] > 0 && l[i] >= l[k-1] {
			count++
		}
	}
	fmt.Println(count)
}

File: codeforces/c/158A.c
#include<stdio.h>

int main(void) {
	int n, k, count=0;
	scanf("%d %d\n", &n, &k);
	int a[n];
	for (int i = 0; i<n; i++) {
		scanf("%d ", &a[i]);
	}
	for (int i = 0; i<n; i++) {
		if (a[i] > 0 && a[i] >= a[k-1]) {
			count++;
		}
	}
	printf("%d\n", count);
	return 0;
}
